subhub/api/webhooks/routes/abstract.py

```diff
@@ -0,0 +1,39 @@
+import logging
+from abc import ABC
+
+import flask
+
+logger = logging.getLogger("abstract_route")
+log_handle = logging.StreamHandler()
+log_handle.setLevel(logging.INFO)
+logformat = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+log_handle.setFormatter(logformat)
+logger.addHandler(log_handle)
```

Comment:
App-wide logging settings are typically done once near the program setup. Only line 6 is needed per file as a result.

Reply:
Line 6 in most libraries/apps looks[ like one of these](https://github.com/udhayprakash/PythonMaterial/blob/0df4477e8ce13af1f9b18c1032f8d5f6718b4bec/python2/12_Logging/11_logging.py), I've regularly used method 2 or 1 described there.